.travis.yml

```diff
@@ -8,5 +8,6 @@ after_success:
 
 deploy:
   provider: heroku
+  app: dziekanat
   api_key: 1afd9691-b8f3-4e97-8be6-fd7ca9e8a060
-  skip_cleanup: true
```

Comment:
ja rozumiem, ze to automatycznie wskazuje ta ikonka? Ale z tego co czytam, to jest napsiane, ze newline MUSI byc na koncu?